h2o-docs/src/product/blog-releases.rst

```diff
@@ -6,253 +6,47 @@ This page houses the most recent major release blog and focused content blogs fo
 Major Release Blogs
 -------------------
 
-H2O Release 3.44
+H2O Release 3.46
 ~~~~~~~~~~~~~~~~
 
-.. image:: /images/blog/rel-3-44.png
+.. image:: /images/blog/rel-3-46.png
 
-We are excited to announce the release of H2O-3 3.44.0.1! We have added and improved many items. A few of our highlights are the implementation of AdaBoost, Shapley values support, Python 3.10 and 3.11 support, and added custom metric support for Deep Learning, Uplift Distributed Random Forest (DRF), Stacked Ensemble, GLM, and AutoML. Please read on for more details.
+We are excited to announce the release of H2O-3 3.46.0.1! Some of the highlights of this major release are that we added custom metric support for XGBoost, allowed grid search models to be sorted with custom metrics, and we enabled H2O MOJO and POJO to work with MLFlow. Several improvements were also made to the Uplift model (like MLI support). Another exciting update is that we now allow GLM models that were previously built to be used to calculate full loglikelihood and AIC. We also focused on fixing security vulnerabilities. Please read on for more details!
 
-AdaBoost (Adam Valenta)
-'''''''''''''''''''''''
+Security patch updates
+''''''''''''''''''''''
 
-We are proud to introduce `AdaBoost <data-science/adaboost.html>`__, an algorithm known for its effectiveness in improving model performance. AdaBoost is particularly notable for its approach in constructing an ensemble of weak learners (typically decision trees) and sequentially refining them to enhance predictive accuracy. It achieves this by assigning higher weights to misclassified data points in each iteration. This emphasizes the correction of errors and ultimately leads to a more precise and robust predictive model. This adaptability and refinement makes AdaBoost a valuable tool in various domains, allowing it to aid in better predictions and informed decision-making.
+H2O should always be deployed behind firewalls and in protected clusters. Many of the reported vulnerabilities don’t apply because of this because they assume that H2O isn’t deployed under any protection. Regardless, there are several vulnerabilities we did fix during this release:
```

Comment:
This phrase

Many of the reported vulnerabilities don’t apply because of this because they assume that H2O isn’t deployed under any protection.

can be rewritten as:

Many of the reported vulnerabilites assume that H2O isn't deployed under any protection.